src/main/java/org/opensearch/plugin/insights/core/service/categorizer/SearchQueryCategorizer.java

```diff
@@ -87,11 +87,6 @@ public void categorize(SearchQueryRecord record) {
         incrementQueryTypeCounters(source.query(), measurements);
         incrementQueryAggregationCounters(source.aggregations(), measurements);
         incrementQuerySortCounters(source.sorts(), measurements);
-
-        if (logger.isTraceEnabled()) {
```

Comment:
Need to figure out where we want to do this. We can do this in the listener. However, we should make sure that the query shape is computed only once.

Reply:
Added changes to do this in the listener